Control/test/lib/mocha-status-service.js

```diff
@@ -156,14 +156,14 @@ describe('StatusService test suite', () => {
 
     it('should successfully retrieve status and info about Kafka that it is not running', async () => {
       const status = new StatusService(config, {}, {});
-      const kafkaStatus = await status.getKafkaStatus(new KafkaConnector(config.kafka));
+      const kafkaStatus = await status.getNotificationStatus(new NotificationService(config.kafka));
```

Comment:
Could you please make sure all `kafka` naming occurrences are updated to avoid future confusion?

```suggestion
      const notificationStatus = await status.getNotificationStatus(new NotificationService(config.kafka));
```